data/presets/amenity/ticket_validator

```diff
@@ -0,0 +1,20 @@
+{
+    "icon": "temaki-vending_machine",
+    "fields": [
+        "ref",
+        "operator",
+    ],
+    "moreFields": [
+        "colour",
+        "covered",
+        "indoor",
+        "level",
+        "support"
+    ],
+    "geometry": [
+        "point"
+    ],
+    "tags": {
+        "amenity": "ticket_validator"
+    },
```

Comment:
I recommend we also provide Name and some search information.
e.g.:
Name: Ticket Validator
Aliases: ticket puncher, punching machine or ticket marker
Terms: ticket, validator, puncher

Reply:
Oh yes, sure, you're right. I forgot the name. Thank you, I added the terms you mentioned and the name now. Do I have to add the aliases as themselves, too?

Reply:
Thanks for doing this promptly. Adding aliases as well, would be a slight improvement, please read more [here](https://github.com/ideditor/schema-builder/issues/3)

Reply:
Yes, thank you. Added them in https://github.com/openstreetmap/id-tagging-schema/pull/166/commits/14f66c248c82af30b7b5195749a00d73d4b2811d